engine/src/scene/bridge.cpp

```diff
@@ -244,12 +207,168 @@ bool SceneBridge::loadFromFile(Assets& assets, const AnyAsset& handle, Stream& s
+            {
+                if (!components.contains(entity))
+                {
+                    continue;
+                }
+                auto [entityName, subHandle] = originalComponentData(assets, handle, name, *type);
+                if (!entityName.empty())
+                {
+                    auto subScene = assets.read<Scene>(subHandle);
+                    if (cubos::core::reflection::compare(*type, components[entity].get(),
+                                                         subScene->blueprint.components()
+                                                             .at(*type)
+                                                             .at(subScene->blueprint.entities().atRight(entityName))
+                                                             .get()))
+                    {
+                        continue;
+                    }
+                }
+                dirty = true;
+                ser.write(components[entity].type(), components[entity].get());
+                entityJson.emplace(type->name().c_str(), ser.output());
+            }
+            for (auto& [type, relations] : scene->blueprint.relations())
+            {
+                if (!relations.contains(entity))
+                {
+                    continue;
+                }
+                for (auto& [other, relation] : relations[entity])
+                {
+                    auto [entityName, otherName, subHandle] =
+                        originalRelationData(assets, handle, name, scene->blueprint.entities().atLeft(other), *type);
+                    if (!entityName.empty())
+                    {
+                        if (auto subScene = assets.read<Scene>(subHandle);
+                            cubos::core::reflection::compare(*type, relation.get(),
+                                                             subScene->blueprint.relations()
+                                                                 .at(*type)
+                                                                 .at(subScene->blueprint.entities().atRight(entityName))
+                                                                 .at(subScene->blueprint.entities().atRight(otherName))
+                                                                 .get()))
+                        {
+                            continue;
+                        }
+                    }
+                    dirty = true;
+                    ser.write(relation.type(), relation.get());
+                    entityJson.emplace((type->name() + "@" + scene->blueprint.entities().atLeft(other)).c_str(),
+                                       ser.output());
+                }
+            }
+            if (dirty)
+            {
+                entitiesJson.emplace(name.c_str(), entityJson);
+            }
+        }
+    }
+    json.push_back({"entities", entitiesJson});
+
+    stream.print(json.dump(2));
```

Comment:
You are dumping json formatted with 2 spaces indentation, but in this PR you formatted the JSON files to be tab-indented.  Make sure to revert the formatting on the json files so that we have something consistent.